calculator.js

```diff
@@ -1 +1,180 @@
-console.log('start')
\ No newline at end of file
+const numbers = document.querySelectorAll('.number')
+const mainResult = document.querySelector('.result')
+const operators = document.querySelectorAll('.operator')
+const equal = document.querySelectorAll('.equal')
+const deleteEqual = document.querySelectorAll('.deleteEqual')
+const dot = document.querySelectorAll('.dot')
+const deleteLast = document.querySelectorAll('.deleteLast')
+
+let result = ''
+let number1 = ''
+let number2 = ''
+let operator = undefined
+
+count = () => {
+  switch (String(operator)) {
```

Comment:
Why converting operator to string? It is not always a stinrg?

Reply:
fixed